phenotrex/structure/records.py

```diff
@@ -19,7 +20,10 @@ def __repr__(self):
 
 @dataclass
 class PhenotypeRecord:
-    """ TODO add docstring """
+    """
+    Structure containing ground truth class values (0 for trait absent, 1 for trait present) for the
+    trait `trait_name` in sample `identifier`.
+    """
```

Comment:
```suggestion
    """
    Ground truth labels of sample `identifier`,
    indicating presence/absence of trait `trait_name`:
      - 0 if trait is absent
      - 1 if trait is present 
    """
```
Mostly for consistency to the above suggestion.